src/components/common/ColonyHome/ColonyHome.tsx

```diff
@@ -37,57 +36,54 @@ const ColonyHome = () => {
     Number(queryDomainIdFilter),
   );
 
-  const memoizedSwitch = useMemo(() => {
-    if (colony) {
-      return (
-        <RoutesSwitch>
-          <Route
-            path={COLONY_EVENTS_ROUTE}
-            element={
+  if (colony) {
```

Comment:
Changing this to a guard clause such as:
```
if (!colony) {
  return null;
}
```
would reduce nesting

Reply:
Nice catch!